doc/release-notes/8.4.md

```diff
@@ -0,0 +1,31 @@
+# Release notes for Gluster 8.4
+
+This is a bugfix release. The release notes for [8.0](8.0.md), [8.1](8.1.md), [8.2](8.2.md) and [8.3](8.3.md)
+contain a listing of all the new features that were added
+and bugs fixed in the GlusterFS 8 stable release.
+
+**NOTE:** 
+- Next minor release tentative date: Week of 20th Apr, 2021
+- Users are highly encouraged to upgrade to newer releases of GlusterFS.
+
+## Highlights of Release
+
+This release contains majorly the bug fixes as described in the issues section.
+
+## Builds are available at 
+
+[https://download.gluster.org/pub/gluster/glusterfs/8/8.4/](https://download.gluster.org/pub/gluster/glusterfs/8/8.4/)
+
+## Issues addressed in this release
+
+Please find the list of issues added to this release below.
+
+- [#2154](https://github.com/gluster/glusterfs/issues/2154) "Operation not supported" doing a chmod on a symlink
+- [#2107](https://github.com/gluster/glusterfs/issues/2107) mount crashes when setfattr -n distribute.fix.layout -v "yes"...
+- [#2067](https://github.com/gluster/glusterfs/issues/2067) Change self-heal-window-size to 1MB by default
```

Comment:
This is a performance improvement, not a bugfix - should be in its own section?

Reply:
Done